packages/SystemsThinkingWorkbench-Core.package/STWConnection.class/instance/update.with..st

```diff
@@ -1,7 +1,7 @@
 updating
 update: anAspect with: anObject
 
-	self isSynchronous
-		ifFalse: [anAspect == #input
-				and: [anObject == self startNode
-						ifTrue: [self addParticle]]].
+	self isSynchronous ifFalse:
+		[anAspect == #input
+			and: [anObject == self startNode
```

Comment:
Missing a `]` i think.

The behavior is correct, but the semantics are ... confusing.